sdk/core/azure-json/src/main/java/com/azure/json/package-info.java

```diff
@@ -2,6 +2,499 @@
 // Licensed under the MIT License.
 
 /**
- * Contains general classes for handling JSON.
+ * <p>The Azure JSON library provides interfaces for stream-style JSON reading and writing. Stream-style reading and
+ * writing has the type itself defines how to read JSON to create and instance of itself and how it writes JSON.
+ * Azure JSON also provides the ability to provide implementations for JSON reading and writing by offering a service
+ * provider interface to load implementations from the classpath, the library provides a default implementation if
+ * one cannot be found on the classpath.</p>
+ *
+ * <h2>Getting Started</h2>
+ *
+ * <p>{@link com.azure.json.JsonSerializable} is the base of Azure JSON, it is the interface that types implement to
+ * provide stream-style JSON reading and writing functionality. The interface has a single implementable method
+ * {@link com.azure.json.JsonSerializable#toJson(com.azure.json.JsonWriter)} that defines how the object will be written
+ * as JSON, to the {@link com.azure.json.JsonWriter}, and an over-writable static method
```

Comment:
'over-writable static method'? I don't understand what that means in the context of Java that doesn't support overriding static methods?